src/applications/resources-and-support/components/SearchResultList.jsx

```diff
@@ -13,10 +11,7 @@ export default function SearchResultList({
 }) {
   return (
     // eslint-disable-next-line jsx-a11y/no-redundant-roles
-    <ul
-      className="usa-unstyled-list vads-u-padding-x--1 desktop-lg:vads-u-padding-x--0"
-      role="list"
```

Comment:
Laura mentioned in DMs in an earlier that we did not need `role="list"` on `<ul>` elements.